*** Starting uWSGI 2.0.17 (64bit) on [Thu Mar  3 10:15:19 2022] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.29) on 25 March 2021 02:06:11
os: Darwin-21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
nodename: C02F90FMMD6V
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/PycharmProjects/cluster_monitor
writing pidfile to /run/costing.pid
open("/run/costing.pid"): No such file or directory [core/utils.c line 3605]
*** Starting uWSGI 2.0.17 (64bit) on [Thu Mar  3 10:15:52 2022] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.29) on 25 March 2021 02:06:11
os: Darwin-21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
nodename: C02F90FMMD6V
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/PycharmProjects/cluster_monitor
writing pidfile to /run/costing.pid
open("/run/costing.pid"): No such file or directory [core/utils.c line 3605]
*** Starting uWSGI 2.0.17 (64bit) on [Thu Mar  3 10:16:27 2022] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.29) on 25 March 2021 02:06:11
os: Darwin-21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
nodename: C02F90FMMD6V
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/PycharmProjects/cluster_monitor
detected binary path: /usr/local/bin/uwsgi
chdir() to /Users/bytedance/PycharmProjects/cluster_monitor
your processes number limit is 5568
your memory page size is 4096 bytes
detected max file descriptor number: 12544
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on [::]:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:62594 (port auto-assigned) fd 3
Python version: 3.9.4 (default, Apr  5 2021, 01:49:30)  [Clang 12.0.0 (clang-1200.0.32.29)]
Python main interpreter initialized at 0x7fe037705950
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 364440 bytes (355 KB) for 4 cores
*** Operational MODE: preforking ***
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 95287)
spawned uWSGI worker 1 (pid: 95289, cores: 1)
spawned uWSGI worker 2 (pid: 95290, cores: 1)
spawned uWSGI worker 3 (pid: 95291, cores: 1)
spawned uWSGI worker 4 (pid: 95292, cores: 1)
spawned uWSGI http 1 (pid: 95293)
--- no python application found, check your startup logs for errors ---
[pid: 95292|app: -1|req: -1/1] ::ffff:127.0.0.1 () {36 vars in 680 bytes} [Thu Mar  3 10:16:40 2022] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95292|app: -1|req: -1/2] ::ffff:127.0.0.1 () {36 vars in 632 bytes} [Thu Mar  3 10:16:40 2022] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95292|app: -1|req: -1/3] ::ffff:127.0.0.1 () {36 vars in 690 bytes} [Thu Mar  3 10:16:44 2022] GET /hello => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (1 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95291|app: -1|req: -1/4] ::ffff:127.0.0.1 () {36 vars in 637 bytes} [Thu Mar  3 10:16:44 2022] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 95293)
worker 4 buried after 1 seconds
worker 3 buried after 1 seconds
worker 2 buried after 1 seconds
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17 (64bit) on [Thu Mar  3 10:17:43 2022] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.29) on 25 March 2021 02:06:11
os: Darwin-21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
nodename: C02F90FMMD6V
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/PycharmProjects/cluster_monitor
detected binary path: /usr/local/bin/uwsgi
chdir() to /Users/bytedance/PycharmProjects/cluster_monitor
your processes number limit is 5568
your memory page size is 4096 bytes
detected max file descriptor number: 12544
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on [::]:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:63119 (port auto-assigned) fd 3
Python version: 3.9.4 (default, Apr  5 2021, 01:49:30)  [Clang 12.0.0 (clang-1200.0.32.29)]
Python main interpreter initialized at 0x7fe6a6304c40
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 364440 bytes (355 KB) for 4 cores
*** Operational MODE: preforking ***
unable to find "application" callable in file app.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 95382)
spawned uWSGI worker 1 (pid: 95384, cores: 1)
spawned uWSGI worker 2 (pid: 95385, cores: 1)
spawned uWSGI worker 3 (pid: 95386, cores: 1)
spawned uWSGI worker 4 (pid: 95387, cores: 1)
spawned uWSGI http 1 (pid: 95388)
--- no python application found, check your startup logs for errors ---
[pid: 95387|app: -1|req: -1/1] ::ffff:127.0.0.1 () {38 vars in 721 bytes} [Thu Mar  3 10:17:48 2022] GET /hello => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95387|app: -1|req: -1/2] ::ffff:127.0.0.1 () {36 vars in 637 bytes} [Thu Mar  3 10:17:48 2022] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95387|app: -1|req: -1/3] ::ffff:127.0.0.1 () {38 vars in 721 bytes} [Thu Mar  3 10:17:48 2022] GET /hello => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (1 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 95387|app: -1|req: -1/4] ::ffff:127.0.0.1 () {36 vars in 637 bytes} [Thu Mar  3 10:17:48 2022] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 95388)
worker 4 buried after 1 seconds
worker 3 buried after 1 seconds
worker 2 buried after 1 seconds
worker 1 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17 (64bit) on [Thu Mar  3 10:18:45 2022] ***
compiled with version: Apple LLVM 12.0.0 (clang-1200.0.32.29) on 25 March 2021 02:06:11
os: Darwin-21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_X86_64
nodename: C02F90FMMD6V
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 12
current working directory: /Users/bytedance/PycharmProjects/cluster_monitor
detected binary path: /usr/local/bin/uwsgi
chdir() to /Users/bytedance/PycharmProjects/cluster_monitor
your processes number limit is 5568
your memory page size is 4096 bytes
detected max file descriptor number: 12544
lock engine: OSX spinlocks
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on [::]:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:63553 (port auto-assigned) fd 3
Python version: 3.9.4 (default, Apr  5 2021, 01:49:30)  [Clang 12.0.0 (clang-1200.0.32.29)]
Python main interpreter initialized at 0x7f92ec705cd0
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 364440 bytes (355 KB) for 4 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x7f92ec705cd0 pid: 95467 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 95467)
spawned uWSGI worker 1 (pid: 95468, cores: 1)
spawned uWSGI worker 2 (pid: 95469, cores: 1)
spawned uWSGI worker 3 (pid: 95470, cores: 1)
spawned uWSGI worker 4 (pid: 95471, cores: 1)
spawned uWSGI http 1 (pid: 95472)
[pid: 95469|app: 0|req: 1/1] ::ffff:127.0.0.1 () {38 vars in 721 bytes} [Thu Mar  3 10:18:52 2022] GET /hello => generated 5 bytes in 4 msecs (HTTP/1.1 200) 2 headers in 78 bytes (1 switches on core 0)
[pid: 95471|app: 0|req: 1/2] ::ffff:127.0.0.1 () {36 vars in 637 bytes} [Thu Mar  3 10:18:52 2022] GET /favicon.ico => generated 232 bytes in 5 msecs (HTTP/1.1 404) 2 headers in 87 bytes (1 switches on core 0)
[pid: 95470|app: 0|req: 1/3] ::ffff:127.0.0.1 () {36 vars in 686 bytes} [Thu Mar  3 10:18:56 2022] GET /job => generated 232 bytes in 3 msecs (HTTP/1.1 404) 2 headers in 87 bytes (1 switches on core 0)
[2022-03-03 10:19:00,450] ERROR in app: Exception on /job/jobs [GET]
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/flask/app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'job_view.get_jobs' did not return a valid response. The function either returned None or ended without a return statement.
[pid: 95471|app: 0|req: 2/4] ::ffff:127.0.0.1 () {36 vars in 696 bytes} [Thu Mar  3 10:19:00 2022] GET /job/jobs => generated 290 bytes in 5 msecs (HTTP/1.1 500) 2 headers in 99 bytes (2 switches on core 0)
